include/install-vitasdk.sh

```diff
@@ -10,30 +10,25 @@ install_vitasdk () {
   case "$(uname -s)" in
      Darwin*)
       mkdir -p $INSTALLDIR
-      wget -O "vitasdk-nightly.tar.bz2" "$(get_download_link master osx)"
-      tar xf "vitasdk-nightly.tar.bz2" -C $INSTALLDIR --strip-components=1
-      rm -f "vitasdk-nightly.tar.bz2"
+      wget -O- "$(get_download_link master osx)" | tar xj -C $INSTALLDIR --strip-components=1
      ;;
 
      Linux*)
       if [ -n "${TRAVIS}" ]; then
           sudo apt-get install libc6-i386 lib32stdc++6 lib32gcc1 patch
       fi
+      command -v curl || echo "curl missing (install using: apt install curl)"
```

Comment:
it seems no more need curl, isn't it?

Reply:
https://github.com/vitasdk/vdpm/blob/1d8138cf27aa4862919ea071a333ac922718ddc2/vdpm#L21
Not sure if it's still used.
If you confirm, I'll remove the line.

Reply:
huh.... i got it...
how about just exit with message before installing?
if we always need the `curl`, it would be acceptable.

Reply:
@d3m3vilurr sure, no problem.
I changed my commit with your solution